cmd/storagenode/main.go

```diff
@@ -173,9 +175,21 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 		"identity.server.address": defaultServerAddr,
 		"storage.path":            filepath.Join(setupDir, "storage"),
 		"kademlia.bootstrap-addr": defaultSatteliteAddr,
+		"log.level":               "debug",
```

Comment:
Why we want to increase to `debug` level?

Reply:
switched to warn